vendor/vendor.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
What generates this file? Perhaps if we're gonna do a `vendor.json` thing  we should do them all at once in a seperate PR?